AvaliadorDisciplinas.FRONTEND/src/components/Dropdown/Dropdown.module.css

```
@@ -0,0 +1,18 @@
+.container {
+  width: 100%;
+  height: 100%;
+}
+
+.container select {
+  width: 100%;
+  height: 100%;
+  border-radius: 1em;
```

Acho que poderia ser:
```suggestion
  border-radius: 1rem;
```

---

Corrigido!